### PR TITLE
Feature: Support export prefix (#337)

### PR DESCRIPTION
**Pull request by skonik**
#### General description
This PR adds `export` prefix  support for `.env` files.
Original proposal topic: https://github.com/dotenv-linter/dotenv-linter/issues/337

##### Before:
```bash
$ echo "export SUPPORT_EXPORT_PREFIX=true" > .env
$ cargo run .
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/dotenv-linter .`
Checking .env
.env:1 IncorrectDelimiter: The export SUPPORT_EXPORT_PREFIX key has incorrect delimiter
.env:1 LowercaseKey: The export SUPPORT_EXPORT_PREFIX key should be in uppercase

Found 2 problems
```

##### After:
```bash
$ echo "export SUPPORT_EXPORT_PREFIX=true" > .env
$ cargo run .
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/dotenv-linter .`
Checking .env

No problems found
```

#### ✔ Checklist:
- [x] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [x] Tests for the changes have been added (for bug fixes / features);
- [x] Docs have been added / updated (for bug fixes / features). dotenv-linter/dotenv-linter.github.io#14

### CODE REVIEW

File: src/common/line_entry.rs [hunk 1]
```diff
@@ -50,6 +50,13 @@ impl LineEntry {
         self.raw_string.trim()
     }
 
+    pub fn stripped_export_string(&self) -> &str {
```
Comment (mgrachev) [inline src/common/line_entry.rs:53]:
```suggestion
   fn stripped_export_string(&self) -> &str {
```
This function shouldn't be public.

**Review by mgrachev — COMMENTED** [review]
@skonik 👋 Thank you for your contribution! 🚀

I have left some comments. Please look at them 👀

**Comment by mgrachev** [comment]
Why this file is needed `bin/dotenv-linter`?

**Comment by mgrachev** [comment]
Also, please add tests for these checks:
* IncorrectDelimiter
* LowercaseKey

**Comment by mgrachev** [comment]
Also, please add information about this feature to [dotenv-linter.github.io](https://github.com/dotenv-linter/dotenv-linter.github.io/blob/master/docs/usage.md).

**Comment by skonik** [comment]
> Also, please add information about this feature to [dotenv-linter.github.io](https://github.com/dotenv-linter/dotenv-linter.github.io/blob/master/docs/usage.md).

Check please https://github.com/dotenv-linter/dotenv-linter.github.io/pull/14

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@skonik Great job! 🦀 Thank you! 👍

**Pull request merged**